app/helpers/object_link_helper.rb

```diff
@@ -146,9 +146,11 @@ def location_search_links(name)
     [
       link_to("Google Maps", "http://maps.google.com/maps?q=" + search_string),
       link_to("Yahoo Maps", "http://maps.yahoo.com/#mvt=m&q1=" + search_string),
-      link_to("Wikipedia",
-              "http://en.wikipedia.org/w/index.php?title=Special:Search&search=" +
-              search_string),
+      link_to(
```

Comment:
I'd make a little helper function, say `search_link` that gets called as `search_link("Google Maps", search_string)` or even `search_link(GOOGLE_MAPS, search_string)` and then have the URLs as part of a constant dictionary where the key is the site name.

Reply:
I like it.